dbms/src/Server/PerformanceTest.cpp

```diff
@@ -667,34 +665,21 @@ class PerformanceTest
         for (const String & precondition : preconditions)
         {
             if (precondition == "reset_cpu_cache")
-                if (system("(>&2 echo 'Flushing cache...') && (sudo sh -c 'echo 3 > /proc/sys/vm/drop_caches') && (>&2 echo 'Flushed.')")) {
+                if (system("(>&2 echo 'Flushing cache...') && (sudo sh -c 'echo 3 > /proc/sys/vm/drop_caches') && (>&2 echo 'Flushed.')"))
+                {
                     std::cerr << "Failed to flush cache" << std::endl;
                     return false;
                 }
 
             if (precondition == "ram_size")
             {
-#if __has_include(<sys/sysinfo.h>)
-                struct sysinfo *system_information = new struct sysinfo();
-                if (sysinfo(system_information))
+                size_t ram_size_needed = config->getUInt64("preconditions.ram_size");
+                size_t actual_ram_mb = getMemoryAmount() / 1024 / 1024;
+                if (ram_size_needed > actual_ram_mb)
```

Comment:
Inconsistent style of sizes in configuration. All sizes should be in number of bytes.